views/scan.ejs

```diff
@@ -405,7 +405,7 @@
 
       if (data.Items.length) {
         $('#items-container').append(data.Items.map(item => {
-          const viewUrl = '/tables/<%= Table.TableName %>/items/' + encodeURIComponent(Object.values(item.__key).join(','))
+          const viewUrl = '/tables/<%= Table.TableName %>/items/' + encodeURIComponent(Object.values(Object.values(item.__key)[0]).join(','))
```

Comment:
I ran this locally for my dataset, and this doesn't work properly when there are multiple keys involved, as this is only picking the first key.
I fixed this locally by doing:
`
Object.values(item.__key).map(function (i) { return Object.values(i)[0] }).join(',')
`